apps/site/lib/site_web/controllers/stop_controller.ex

```diff
@@ -444,4 +455,8 @@ defmodule SiteWeb.StopController do
 
     %{conn | assigns: assigns}
   end
+
+  defp route_sorter(%{route: route}) do
+    {Routes.Group.sorter({Route.type_atom(route), nil}), route.sort_order}
```

Comment:
Tiny thing: `Routes.Group` has an alias, so this could just be `Group.sorter`.

Reply:
There's an existing `grouped_routes` function in this module that calls `Group.sorter`. Would we want to update that to use this better sorter as well?

Reply:
I tried, it was a bit weird given that that function also groups the results. Ultimately I decided I might not need to have a new `route_sorter` function now that I have a `sort_order` attribute to use!